site/en/blog/new-in-chrome-89/index.md

```diff
@@ -0,0 +1,223 @@
+  more!
+layout: 'layouts/blog-post.njk'
+date: 2021-03-02
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/st3dNRCY5FpDmmZXJ8ZG.jpg'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-89
+---
+
+Chrome 89 is starting to roll out to stable now.
+
+{% YouTube id='O-gv6zLZHKw' %}
+
+Here's what you need to know:
+
+* [WebHID, WebNFC, and Web Serial](#hardware) have graduated from their
+  origin trials and are now available in stable.
+* We're closing a loophole a few developers used to skirt the
+  [PWA installability checks](#installability).
+* [Web Share, and Web Share Target](#webshare) arrive on the desktop.
+* And, there's [plenty more](#more).
+
+I'm [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 89!
+
+## WebHID, WebNFC, and Web Serial {: #hardware }
+
+I'm really excited about WebHID, WebNFC, and Web Serial. They open up new
+scenarios for users that were never possible before, interacting with real
+world hardware.
+
+It allows makers to connect to fun, quirky hardware, video conferencing apps
+to use the dedicated telephony buttons on specialized speakers. Or any number
+of other use cases.
+
+{% Video className="float-right", autoplay="true", muted="true", loop="true", src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/g8VCbIMljnxuemDFxl99.mp4" %}
+
+Using Web Serial, and [about 60 lines of code][pico-code],
+[@AndreBan][andre-tweet] created a [page][andre-page] that can interact with
+the MicroPython REPL on a [Raspberry Pi Pico][rpi-pico]. Web Serial is also
+used by Espruino in their [web based IDE][espruino-ide].
+
+At CDS last year, Francois wrote a fun, memory-style game using Web NFC.
+You had to tap the phone to the right card, in the right order.
+
+<div style="clear:both;"></div>
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/VAuLti7I9ZQCwLUEWF5Y.png", className="float-right", alt="StreamDeck with Daft Punk Drum Pad", width="800", height="536" %}
+
+And my favorite, [@bramus][bramus-tweet] used WebHID to connect to a
+StreamDeck, building a [Daft Punk drum pad][bramus-demo]. If you don't have
+a StreamDeck, check out his [demo video][bramus-demo-video] on YouTube, and
+check out the [code on GitHub][bramus-code].
+
+Whether it's your site, that interacts with your hardware, or your hardware
+that can interact with lots of sites, users win because they don't need to
+install special drivers, or software.
+
+You can learn more about some of the devices you can connect to at
+[web.dev/devices](https://web.dev/devices/), or check out the getting started
+guides for [WebHID](https://web.dev/hid/), [WebNFC](https://web.dev/nfc/),
+and [Web Serial](https://web.dev/serial/).
+
+## PWA installability criteria changes {: #installability }
+
+Offline support has been a key part of the Progressive Web App criteria for
+installability since the beginning. Like any other installed app, users
```

Comment:
```suggestion
installability since the beginning. As with other installed app, users
```

Reply:
I know this looks like the old like/as confusion. Actually what you have is a dangling modifier that makes it sound like a user is an installed app.